exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/trace_exporter/__init__.py

```diff
@@ -97,6 +101,12 @@ def __init__(
             int(environ_timeout) if environ_timeout is not None else None
         )
 
+        compression = (
+            environ_to_compression(OTEL_EXPORTER_OTLP_SPAN_COMPRESSION)
```

Comment:
Isn't it supposed to be `*_TRACES_COMPRESSION`? (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options)

Reply:
Nice catch 👍

Reply:
Renamed all of those env vars in 69f695a3